product_docs/docs/postgres_distributed_for_kubernetes/1/known_issues.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: 'Known issues'
+---
+
+This is a currently known issue in EDB Postgres Distributed for Kubernetes. 
```

Comment:
should this be plural? "These are currently known ..." ?

Reply:
Yeah, something like "The following is a list of currently known issues..."

Reply:
Any maybe mention issues AND work arounds (where possible).